package.json

```diff
@@ -1,20 +1,24 @@
 {
   "name": "teleport-ui",
   "version": "1.0.0",
-  "main": "index.js",
   "scripts": {
     "build-ui": "yarn build-ui-oss && yarn build-ui-e",
     "build-ui-oss": "yarn workspace @gravitational/teleport build --output-path=../../../webassets/teleport/app",
     "build-ui-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../webassets/e/teleport/app",
-    "build-teleport": "yarn workspace @gravitational/namespaces build-teleport",
-    "start-teleport": "yarn workspace @gravitational/namespaces start-teleport",
-    "build-term": "yarn workspace @gravitational/namespaces build-term",
-    "start-term": "yarn workspace @gravitational/namespaces start-term",
-    "package-term": "yarn workspace @gravitational/namespaces package-term",
-    "build-native-deps-for-term": "yarn workspace @gravitational/namespaces build-native-deps-for-term",
-    "build-and-package-term-ci": "yarn workspace @gravitational/namespaces build-and-package-term-ci",
+    "build-teleport": "yarn build-teleport-oss && yarn build-teleport-e",
```

Comment:
Should this call `build-ui` and then `build-teleport-e` and `build-teleport-oss` can go?

Reply:
yes plz :) This was my intention with the `build-ui`, `build-ui-oss`, and `build-ui-e` naming scheme.

Reply:
Should we just remove all three `build-teleport*` scripts completely then? I don't see them being referenced anywhere, other than `e/web/README.md`.

Reply:
Yeah sure that sounds good